### DISCUSSION TITLE
Reorder Images in Carousel Block

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#1185] [Q&A] [answered]
How do you reorder images in the carousel block? I'm looking at the RC2 and I'm not seeing anything on how to change the order of the images, but it seems that #873 has been closed. Do you have a quick screenshot or video of what I'm missing?

**Comment by meetmandeep** [comment, marked as answer]
Use the navigator: 
![image](https://user-images.githubusercontent.com/29958123/117365756-9f7ada00-ae74-11eb-8501-11884957b8ce.png)

**Reply by joshgk00** [reply]
Ah, makes sense. Thanks!
